### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by gregoryjflaherty**
Add gems
Add PR template 
Create models for users, exercises, workouts, sets
Create database and schema

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2022_05_13_204354) do
```
Comment (SullyBirashk) [inline db/schema.rb:13]: Great Schema!

File: db/migrate/20220513204354_create_sets.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateSets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :sets do |t|
+      t.integer :reps
+      t.float :weight_lbs
```
Comment (samlsmith424) [inline db/migrate/20220513204354_create_sets.rb:5]: Clean, concise, and user friendly with the float option for weights!

**Review by SullyBirashk — APPROVED** [review]
Set up looks Great!

**Comment by samlsmith424** [comment]
Great job the with initial set up!

**Pull request merged**